olive/passes/pytorch/lora.py

```diff
@@ -454,14 +454,14 @@ def init_lora_adapters(
                 "loftq_config": LoftQConfig(loftq_bits=4, loftq_iter=config.loftq_iter),
             }
 
-        peft_task_type = get_peft_task_type_from_task(task, fail_on_not_found=True)
+        peft_task = get_peft_task_from_task(task, fail_on_not_found=True)
         lora_config = LoraConfig(
             r=config.lora_r,
             lora_alpha=config.lora_alpha,
             lora_dropout=config.lora_dropout,
             target_modules=target_modules,
             bias=config.bias,
-            task_type=peft_task_type,
+            task=peft_task,
```

Comment:
same. Still task type. I think we can just keep the task_type name and code the same. Just need to update the config json with the correct name.

Reply:
like in https://github.com/microsoft/Olive/pull/928

Reply:
even if we want to update the function names etc, we could do it a separate PR to avoid overloading this PR

Reply:
I as just thinking it is about renaming, then updated it. You are right. Reverted the changes, let us update in following pr.

Reply:
Also saw your PR, so I just updated the bert_snpe.json.